Formula/etcher-cli.rb

```diff
@@ -0,0 +1,26 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher.git",
+    :tag => "v1.4.4",
+    :revision => "cbd531e161a12721fb763b526781f66ef8237ac9"
+
+  depends_on "python" => :build
+  depends_on "jq"
+  depends_on "node"
+
+  def install
+    Language::Node.setup_npm_environment
+    system "make", "cli-develop", "RELEASE_TYPE=production"
+    system "make", "package-cli", "RELEASE_TYPE=production"
+    inreplace "Makefile", "	mocha", "npx -p node@6	mocha"
```

Comment:
mocha is called globally, which does not inherit the node bin context - so we specify node@6 using npx. It would be nice to bail early on install if npx is not in the global path, but i'm not sure the best way to test this, since npx could be installed not to the homebrew bin folder.